### DISCUSSION TITLE
Tool tip header custom date

### DISCUSSION THREAD

**Discussion by eversadia** [TanStack/react-charts#257] [Q&A]
Is there a way for the tool tip header to be customize some thing like the image below

![line-chart-header](https://user-images.githubusercontent.com/7265819/140015281-50be21ac-126b-4ee4-bcd9-b2f7633c025a.jpg)

**Comment by konstantinkreft** [comment]
Yep, that's definitely possible. You can add a tooltip formatter to to the `primaryAxis` to achieve this.

```ts
  const primaryAxis = React.useMemo<
    AxisOptions<typeof data[number]["data"][number]>
  >(
    () => ({
      getValue: (datum) => (datum.primary as unknown) as Date,
      formatters: {
        tooltip: (value: Date | null) => {
          if (!value) return "";
          return value.toLocaleString();
        }
      }
    }),
    []
  );
```

Added it to a codesandbox so you can tinker around a bit: https://codesandbox.io/s/crimson-rain-4m3fe?file=/src/components/Line.tsx

**Reply by ananduSiva** [reply]
Is there any documentation for other options apart from getValue and formatters ? Only getValue is mentioned in the official documentation.

Thank you for your help in advance
